### ISSUE TITLE
Add molql string interpolation support to label representation

### ISSUE THREAD

**Issue by arose** [molstar/molstar#330]
Issue #313 asks for support to add a formal charge label to a representation. I suggest a more general solution to this for the built-in label representation.

We could add a string parameter that accepts molql symbols, for example, `'${atom.pdbx_formal_charge}'` or `'My Atom: ${atom.label_atom_id}'`. Using molql symbols has the benefit that custom properties are supported, e.g., `ma.quality-assessment.pLDDT` from `extensions/model-archive/quality-assessment/prop.ts`. The molql symbols would be evaluated for each atom a label is needed.

**Comment by arose** [comment]
ping @dsehnal @ptourlas

**Comment by dsehnal** [comment]
This sounds like a good idea. Could be implemented as a behavior with a list of label specs as params to support multiple labels like this.

**Comment by arose** [comment]
> Could be implemented as a behavior with a list of label specs as params to support multiple labels like this.

I don't understand what you mean by that. I was thinking to allow the user to provide a string param to `mol-repr/structure/visual/label-text.ts` that includes one or more molql symbols.

**Comment by dsehnal** [comment]
> I don't understand what you mean by that. I was thinking to allow the user to provide a string param to `mol-repr/structure/visual/label-text.ts` that includes one or more molql symbols.

Right, I was thinking we could add the same support for hover labels too.

**Comment by arose** [comment]
> Right, I was thinking we could add the same support for hover labels too.

yes, nice idea
